controllers/bridge.go

```diff
@@ -0,0 +1,24 @@
+package controllers
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"github.com/tree-graph/bridge-service/infra/blockchain"
+	"github.com/tree-graph/bridge-service/models"
+)
+
+func CrossRequest(ctx *gin.Context) (interface{}, error) {
+	bean := new(models.CrossRequest)
+
+	err := ctx.ShouldBindJSON(&bean)
+	if err != nil {
+		logrus.WithError(err).Error("bind json error")
```

Comment:
suggest to use `debug` level rather than `error` level since this is due to invalid parameters passed by client.